packages/table/src/Centered.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+const displayFlex = css`
+  display: flex;
+  flex-grow: 1;
+`;
+
+const centeredHorizontal = css`
+  justify-content: center;
+`;
+
+const centeredVertical = css`
+  align-items: center;
+`;
+
+interface CenteredProps {
+  children: React.ReactNode;
+  horizontal?: boolean;
+  vertical?: boolean;
+}
+
+function Centered({
+  children,
+  horizontal = false,
+  vertical = false,
```

Comment:
Hm, would it be counter-intuitive for neither prop being passed in `<Centered>` to default to centering both? just as an example of how this logic might look
```jsx
<Centered horizontal> or <Centered vertical={false}> // center horizontal
<Centered vertical> or <Centered horizontal={false}> // center vertical
<Centered> or <Centered horizontal vertical> // center both
```

Reply:
yeah actually i think you're right. if it's centered then the default behavior should be to center it